docs/modules/quickstarters/pages/be-rust-axum.adoc

```diff
@@ -190,11 +193,11 @@ The Jenkinsfile is provisioned with this quick starter to ease CI/CD process. In
 
 include::partials$secret-scanning-with-gitleaks.adoc
 
-== Builder agent used
+== Builder agent used by the Jenkins CICD
```

Comment:
typically we use CI/CD, no?

Reply:
yes, that is implicit as stated in all the other quickstarters, if we want to change it here we shall change it everwhere, I close this as I don't see it required and nevertheless could be tackled in another PR later, ok?